pkg/cas/resolver/resolver.go

```diff
@@ -44,7 +55,7 @@ func New(casClient casapi.Client, httpClient *http.Client) *Resolver {
 //    Finally, the data is returned to the caller.
 // In both cases above, the CID produced by the local CAS will be checked against the cid passed in to ensure they are
 // the same.
-func (h *Resolver) Resolve(webCASURL *url.URL, cid string, data []byte) ([]byte, error) {
+func (h *Resolver) Resolve(webCASURL *url.URL, cid string, data []byte) ([]byte, error) { //nolint:gocyclo,cyclop
```

Comment:
From the code below, it looks like `cid` passed in here will contain the hint (domain + either `ipfs` or `webcas`). So does this mean that when I enable WebFinger, we won't need that `webCASURL` parameter?

Reply:
(I wonder if you still need that `webCASURL` in this PR, even?)

Reply:
You still need it, since you may not have hint.

Reply:
So you can have WebCASURL and `cid` without hint? Does that hint-less `cid` string come from ActivityPub?

Reply:
yes, it can come from other sources as well (as you can see our last resource is IFPS)